extensions/oidc-client-filter/runtime/src/main/java/io/quarkus/oidc/client/filter/runtime/AbstractOidcClientRequestFilter.java

```diff
@@ -21,6 +21,10 @@ public AbstractOidcClientRequestFilter() {
 
     @Override
     public void filter(ClientRequestContext requestContext) throws IOException {
+        if (isClientFeatureDisabled()) {
+            LOG.debug("OIDC client feature is disabled with `quarkus.oidc-client.enabled=false`.");
```

Comment:
I think at this place, debug message requires bit more context. You tell users it is disabled, but you don't say what is happening there. Would it make sense to add sentence saying no token propagation is going to happen?